ext/standard/var_unserializer.re

```diff
@@ -438,6 +451,16 @@ static int php_var_unserialize_internal(UNSERIALIZE_PARAMETER, int as_key);
 
 static zend_always_inline int process_nested_data(UNSERIALIZE_PARAMETER, HashTable *ht, zend_long elements, zend_object *obj)
 {
+	if (var_hash) {
+		if ((*var_hash)->max_depth > 0 && ++(*var_hash)->cur_depth > (*var_hash)->max_depth) {
+			php_error_docref(NULL, E_WARNING,
+				"Maximum depth of " ZEND_LONG_FMT " exceeded. "
+				"You can change the depth limit using the max_depth option",
```

Comment:
I'm not sure what the policy on error messages is, but wouldn't it be better to have something akin to ``The depth limit can be modified by using the max_depth option`` ?

Reply:
Agree and done.